### DISCUSSION TITLE
fastlane plugin for amazon appstore

### DISCUSSION THREAD

**Discussion by felixwortmann** [fastlane/fastlane#19009] [Ideas]
Since Windows 11 was announced to support Android apps from the amazon appstore, I guess the demand for a fastlane plugin for the amazon appstore is bigger than ever. I saw #1963 and #15148, which are both closed by now. Is there anything happening on this topic? Can we expect this feature to be supported at some point?

**Comment by sp71** [comment]
I would also be interested

**Comment by Moodyan16** [comment]
Thanks for the information, I will try to figure it out for more. Keep sharing such informative updates. 

[MyMileStoneCard](https://www.mymilestonecard.org/)

**Reply by jcsahnwaldt** [reply]
SPAM
